compiler/damlc/tests/BUILD.bazel

```diff
@@ -1001,3 +1001,17 @@ daml_compile_with_dalf(
     # PlatformIndependence.daml.
     ghc_options = [],
 )
+
+# Test for interfaces + data-dependencies
+daml_build_test(
```

Comment:
It seems slightly unintuitive that we test this here instead of in `//compiler/damlc/tests:data-dependencies`. I totally get wanting to work with files instead of string literals but I wonder if we’re better off keeping things consistent for now and having the data-dependencies tests all together instead of building a bunch of different test suites for different parts of data-dependencies.

Reply:
hm yeah that's a good point. I guess since most of the work is done and the files probably don't need to change much I can move it to `//compiler/damlc/tests:data-dependencies` more easily